### PR TITLE
Sync Quantity and Shipping Simulator

### PR DESCRIPTION
**Pull request by lucis**
https://github.com/vtex-sites/marinbrasil.store/pull/172

### CODE REVIEW

**Review by klzns — COMMENTED** [review]
How can one test a PR? Does it have a link to a build preview?

**Comment by lucis** [comment]
Sorry, @klzns, it was supposed to be a draft. I'll add the Deploy Preview later

File: packages/gatsby-theme-vtex/src/components/ShippingSimulator/hooks/useShippingSimulator.ts [hunk 1]
```diff
@@ -13,13 +13,15 @@ type HookProps = {
   skuId: string
   seller: string
   country: string
+  quantity: string
```
Comment (tlgimenes) [inline packages/gatsby-theme-vtex/src/components/ShippingSimulator/hooks/useShippingSimulator.ts:16]: Why quantity is a string ?

**Review by tlgimenes — APPROVED** [review]
LGTM for me, however just left a question

**Pull request merged**